### ISSUE TITLE
Inconsistent identification of files with no labels

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#5287]
When a file has been added without a label - in some places the label is then missing, leaving the file with no identifier, in others 'no label' is specifically stated, and in yet others, the filename is used so there is an identifier, but it is not obvious the label is missing.

Files should be consistently identified.  

This within the spirit of WCAG 3.2.4 which is about consistently identifying functions.  While files aren't functions, it is equally unhelpful having files identified differently in different places when we are handling workflows with multiple files.

### Article In Review
Left has label, right has no label

<img width="786" height="277" alt="Image" src="https://github.com/user-attachments/assets/1d49eb0d-9a34-4e84-b763-6fabebf89bd2" />

### Document Management

<img width="904" height="219" alt="Image" src="https://github.com/user-attachments/assets/618c272d-e067-4744-8a5c-2f9b536da1f3" />

### Review Assignment

<img width="824" height="241" alt="Image" src="https://github.com/user-attachments/assets/7497df47-855c-4b5d-8535-3e1924c08db2" />
